controller/planner_backlog_test.go

```diff
@@ -72,11 +72,11 @@ func (rest *TestPlannerBacklogREST) setupPlannerBacklogWorkItems() (testSpace *s
 		_, err := spacesRepo.Create(rest.ctx, testSpace)
 		require.Nil(rest.T(), err)
 		require.NotNil(rest.T(), testSpace.ID)
-		logrus.Info("Created space with ID=", testSpace.ID)
+		log.Logger().Infoln("Created space with ID=", testSpace.ID)
```

Comment:
In any case, we could use one or the other that depends on what you want to log.